### PR TITLE
igork17@gmail.com - Action<T> & Func<T, TResult> delegates samples.

### PR DESCRIPTION
**Pull request by korolei**

### CODE REVIEW

File: csharp-advanced-topics/ActionFuncInCSharp/ActionFuncInCSharp/ActionFuncInCSharp.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/ActionFuncInCSharp/ActionFuncInCSharp/ActionFuncInCSharp.csproj:6]: No global usings in .NET 5

File: csharp-advanced-topics/ActionFuncInCSharp/ActionFuncInCSharp/NotifyMe.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace ActionFuncInCSharp;
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/ActionFuncInCSharp/ActionFuncInCSharp/NotifyMe.cs:1]: File scoped namespaces, .NET 6

File: csharp-advanced-topics/ActionFuncInCSharp/ActionFuncInCSharp/Program.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿namespace ActionFuncInCSharp;
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/ActionFuncInCSharp/ActionFuncInCSharp/Program.cs:1]: File scoped namespaces, .NET 6, missing references

File: csharp-advanced-topics/ActionFuncInCSharp/new 3.txt [hunk 4]
```diff
@@ -0,0 +1,25 @@
+nuget.exe push -Source "CAT-OS_Feed" -ApiKey az "C:\Users\IgorKorolenko\source\repos\CAT-OS-Project\*.nupkg" --skip-duplicate
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/ActionFuncInCSharp/new 3.txt:1]: What is this file for?

**Review by CodeMazeBlog — COMMENTED** [review]
Overall, missing references because of the implicit using, file scoped namespaces. Maybe open in VS2019 and build locally. When you submit the changes, check if the Jenkins build is okay please.

**Comment by CodeMazeBlog** [comment]
retest this please

File: csharp-advanced-topics/ActionFuncInCSharp/new 3.txt (hunk 4 above)
Comment (korolei) [reply csharp-advanced-topics/ActionFuncInCSharp/new 3.txt:1]: This should be excluded. It's for other project.

**Comment by korolei** [comment]
I'm closing this PR, as solution needs to be recreated in .NET Framework 4.8 to run successfully here.

**Comment by CodeMazeBlog** [comment]
It can run as a .NET 5 project. You don't need to recreate it. Jenkins has the latest stable .NET SDK which is .NET 5 currently.

**Pull request closed without merging**